src/app/downloads/downloads.component.ts

```diff
@@ -27,6 +32,7 @@ export class DownloadsComponent implements OnInit, OnDestroy {
   isEpisodeLoading = true;
   error: string;
 
+
```

Comment:
the [commit message](https://github.com/PRX/metrics.prx.org/pull/30/commits/e9e74b2ea5091469aff7bdbd689117d410378c6b) for this line addition was `reset loading spinner when making new castle requests` -- i think maybe something got lost?

Reply:
I think what happened was those changes actually got merged into the previous PR and things got mixed up with what was being committed here. Too many branches branched form each other problem.

Reply:
👍